discovery/kubernetes/kubernetes.go

```diff
@@ -136,11 +139,12 @@ func (c *SDConfig) UnmarshalYAML(unmarshal func(interface{}) error) error {
 
 	foundSelectorRoles := make(map[Role]struct{})
 	allowedSelectors := map[Role][]string{
-		RolePod:      {string(RolePod)},
-		RoleService:  {string(RoleService)},
-		RoleEndpoint: {string(RolePod), string(RoleService), string(RoleEndpoint)},
-		RoleNode:     {string(RoleNode)},
-		RoleIngress:  {string(RoleIngress)},
+		RolePod:           {string(RolePod)},
+		RoleService:       {string(RoleService)},
+		RoleEndpointSlice: {string(RolePod), string(RoleService), string(RoleEndpointSlice)},
```

Comment:
```suggestion
		RoleEndpointSlice: {string(RolePod), string(RoleService), string(RoleEndpointSlice)},
```
Are you sure about this one?

Reply:
Yes I'm sure the EndpointSlice role should have EndpointSlice selector. Maybe I'm misunderstanding your question. Can you be more specific?

Reply:
See my other question :)